discussion/app/controllers/CommentsController.scala

```diff
@@ -46,10 +46,10 @@ object CommentsController extends DiscussionController with ExecutionContexts {
         Cached(60) {
           if (request.isJson)
             JsonComponent(
-              "html" -> views.html.fragments.comment(comment).toString
+              "html" -> views.html.fragments.comment(comment, comment.discussion.isClosedForRecommendation).toString
             )
           else
-            Ok(views.html.fragments.comment(comment))
+            Ok(views.html.fragments.comment(comment, comment.discussion.isClosedForRecommendation))
```

Comment:
`isClosedForRecommendation` default value is `false` 
https://github.com/guardian/frontend/blob/master/discussion/app/views/fragments/comment.scala.html#L1